components/service/glean/scripts/sdk_generator.gradle

```diff
@@ -57,10 +57,11 @@ preBuild.finalizedBy("build_envs")
 
 // Generate the Metrics API
 ext.gleanGenerateMetricsAPI = {
-    namespace -> {
+    {
         variant ->
 
         def sourceOutputDir =  "$buildDir/telemetry/src/${variant.name}/kotlin"
+        def fullNamespace = "${variant.applicationId}.metrics"
```

Comment:
I'd rather have the sob package be more specific to avoid problems, e.g `GleanMetrics`

Reply:
I agree with Dexter here, that glean should be in the sub-package name.  It's also good advertising :)

Reply:
Sure.  Done.